apps/graphql/src/dataloaders/itinerariesHelpers.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import type {
+  ApiRouteItemType,
+  VehicleTypeType,
+} from './ItinerariesloaderTypes';
+
+export const getVehicle = (type: ?VehicleTypeType, uniqueNo: string) => ({
+  type: type ?? null,
+  uniqueNo: uniqueNo ?? null,
+});
+
+export const getTransporter = (transporter: ?string) => ({
+  name: transporter ?? null,
+});
+
+export const getItineraryType = (routes: ?Array<Array<string>>) => {
+  if (routes == null) {
+    return null;
+  }
+  if (routes.length === 1) {
+    return 'oneway';
+  }
+  if (routes.length === 2) {
+    return 'return';
+  }
+  return null;
+};
+
+export const getSectors = (
+  routesList: Array<ApiRouteItemType>,
+  routesMap: Array<Array<string>>,
+): Array<Array<ApiRouteItemType>> => {
+  let routesSubList = routesList;
+  return routesMap.map(routeMap => {
+    const routeListEndSectorIndex = routesSubList.findIndex(
+      routeItem => routeItem.flyTo === routeMap[1],
+    );
+    const tempRoutesSubList = routesSubList;
+    routesSubList = tempRoutesSubList.slice(routeListEndSectorIndex + 1);
+    return tempRoutesSubList.slice(0, routeListEndSectorIndex + 1);
+  });
+};
+
+export const getDate = (local: ?Date, utc: ?Date) => ({
+  local: local ?? null,
+  utc: utc ?? null,
+});
+
+export const getCountry = (name: ?string, code: ?string) => ({
+  id: code ?? '',
+  name: name ?? '',
+  code: code ?? '',
```

Comment:
Shouldn't we discussed that we want to return `null` instead of empty string? 
Same with `getLocation`

Reply:
Done